serving/docker/config.properties

```diff
@@ -1,5 +1,6 @@
 inference_address=http://0.0.0.0:8080
 management_address=http://0.0.0.0:8080
+cluster_address=http://0.0.0.0:8888
```

Comment:
Nit: any reason for choosing 8888? Should it be a closer port like 8081?

Reply:
User may choose management port as 8081